src/content/handbook/management/people-management.md

```diff
@@ -22,7 +22,7 @@ As a people manager, you would be expected to:
 - support the people you manage to access the right training, including apprenticeships and certifications
 - act as the primary contact for people you manage for all [people queries and issues](https://intranet.justice.gov.uk/guidance/hr/), including pay and expenses,leave, conduct and behaviour, absence management, and performance management
 - hold regular conversations with the people you manage
-- help the people you manage with their DDaT assessments, and ensure that they are getting the right opportunities to meet the criteria
+- help the people you manage with their Government Digital and Data assessments, and ensure that they are getting the right opportunities to meet the criteria
```

Comment:
Is there a link we can add for Government Digital and Data assessments? It's the first time it's mentioned in the handbook and might benefit for some context